### PR TITLE
refactor: undo renovate front-end updates

### PR DESCRIPTION
**Pull request by OmranAbazid**
## Description
See https://camunda.slack.com/archives/C06UJTKV60N/p1723457462941119
Renovate is updating packages automatically which already broke the front-end unit test ci job.
This PR is to undo the updates. We will address the automatic updates in other issues.

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #

### CODE REVIEW

File: optimize/client/src/components/Home/Collection.test.js [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 import React, {runAllEffects} from 'react';
 import {shallow} from 'enzyme';
+import {Tag} from '@carbon/react';
```
Comment (misiekhardcore) [inline optimize/client/src/components/Home/Collection.test.js:11]: why do we need to do this now?

**Review by misiekhardcore — APPROVED** [review]
LGTM, just one question

**Comment by npepinpe** [comment]
If the Renovate updates are causing you problems, why not disable Renovate for Optimize for now?

**Comment by misiekhardcore** [comment]
@npepinpe we don't want to disable renovate cause we don't want to cause any potential vulnerabilities because of that.
Instead we are going to update the renovate config to disable some packages we know we are not going to update anytime soon ([this PR](https://github.com/camunda/camunda/pull/21060)) and we are going to move optimize unit tests to the unified CI workflow ([this PR](https://github.com/camunda/camunda/pull/21063)). This way we reduce the chance of breaking anything in optimize with deps updates and reduce the chance of vulnerabilities popping up if we do not update these deps at all.

**Comment by misiekhardcore** [comment]
@OmranAbazid there are some issues with merging this PR. Maybe rebasing it to `main` will help

**Pull request merged**